packages/components/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -29,6 +29,16 @@ const DList = styled.dl<DListProps>`
         `}
       }
     `}
+  ${({ direction }) =>
+    direction &&
+    direction === 'row' &&
+    css`
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      column-gap: ${list.rowDirection.columnGap};
+      row-gap: 0;
+    `}
```

Comment:
`display: flex` kan vi sikkert ha som default uansett, så er det bare `flex-direction` som blir overstyrt av `direction`.

```suggestion
    display: flex;
    flex-direction: ${({direction = 'column'}) => direction};
    flex-wrap: wrap;
    column-gap: ${list.rowDirection.columnGap};
    row-gap: 0;
```

Reply:
Fixed